easybuild/easyblocks/c/cp2k.py

```diff
@@ -569,7 +569,14 @@ def configure_ACML(self, options):
 
     def configure_BLAS_lib(self, options):
         """Configure for BLAS library."""
-        options['LIBS'] += ' %s %s' % (self.libsmm, os.getenv('LIBBLAS', ''))
+
+        flexiblas_root = get_software_root('FlexiBLAS')
+        if flexiblas_root:
+            self.log.debug('Using FlexiBLAS as BLAS lib')
+            options['LIBS'] += ' %s %s' % (self.libsmm, '-lflexiblas -lgfortran')
+            print('LIBS:', options['LIBS'])
```

Comment:
@smoors This `print` statement is probably debug stuff? Should be cleaned up imho...

Reply:
Isn't it a bit strange to do this `FlexiBLAS` but not for any other BLAS library?

Reply:
we can do it for other BLAS libs as well, but then it should be implemented differently, because in that case there is no way to decide which one should be used.

should I add an extra_vars parameter?

Reply:
I wonder if it's worth the trouble... What's the use case you have for this exactly? Is it just to play with FlexiBLAS without having to use a toolchain that uses it?

Reply:
>Is it just to play with FlexiBLAS without having to use a toolchain that uses it?

yes, exactly.
I wondered too, that's why I posted this, to see what other people think of it.
no worries, I'll just close this and post it on the blis-eval repo.